src/DependencyInjection/Configuration.php

```diff
@@ -156,7 +156,20 @@ private function createClientsNode() : ArrayNodeDefinition
                                     ->thenInvalid('cert can be: string or array with two entries (path and password)')
                                 ->end()
                             ->end()
-                            ->floatNode('connect_timeout')->end()
+                            ->scalarNode('connect_timeout')
+                                ->example('%env(float:CONNECT_TIMEOUT)%')
```

Comment:
I don't know if it's good example. This option can be used without env too. For example `connect_timeout: 1`. I suggest to remove this example. Who want to use it with env, will use it.

Reply:
Done

Reply:
Approve from my side. Let's wait approve from @florianpreusner